components/director/internal/domain/labeldef/resolver.go

```diff
@@ -134,8 +134,9 @@ func (r *Resolver) LabelDefinition(ctx context.Context, key string) (*graphql.La
 	}
 
 	if def == nil {
-		return nil, fmt.Errorf("label definition with key '%s' does not exist", key)
+		return nil, nil
```

Comment:
Please notify UI team about our breaking change in API - we will always return null for all queries for single items